projects/hslayers/src/components/save-map/public-api.ts

```diff
@@ -1,16 +1,16 @@
-export * from './dialog-spawner.service';
-export * from './layer-synchronizer.service';
-export * from './layman-layer-descriptor.interface';
+export * from './feature-services/dialog-spawner.service';
```

Comment:
I'm not a fan of feature-services directory. Reminds me of having 'partials' directory which was a mistake. Better to group things by function, not form.